roles/foreman_proxy_certs_generate/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Set tarball path
+  set_fact:
+    foreman_proxy_certs_generate_tarball: /root/{{ foreman_proxy_certs_generate_fqdn }}.tar.gz
+  when: foreman_proxy_certs_generate_tarball is undefined
```

Comment:
I think you can put that into defaults, then you'd save a `set_fact` and that ugly `when` XD

Reply:
:+1: The first thing i learned with ansible is "avoid set_fact". You can never use `--start-at-task` if you have it.

Reply:
Let me translate that for you: "never use set_fact OR start-at-task" ;-)